lib/containers/k8s.pm

```diff
@@ -103,6 +103,8 @@ sub install_k3s {
         delete $k3s_args{$key};
     }
 
+    zypper_call('in k3s-selinux') if is_tumbleweed;
```

Comment:
FWICT the get.k3s.io script would install k3s-selinux itself, so maybe this only needs to be done in the case below, as part of `@pkgs`?

Reply:
> FWICT the get.k3s.io script would install k3s-selinux itself, so maybe this only needs to be done in the case below, as part of `@pkgs`?

See https://github.com/k3s-io/k3s/issues/10876

The script fails with:

```
# zypper in https://rpm.rancher.io/k3s/stable/common/microos/x86_64/
Loading repository data...
Reading installed packages...
'https://rpm.rancher.io/k3s/stable/common/microos/x86_64/' not found in package names. Trying capabilities.
No provider of 'https://rpm.rancher.io/k3s/stable/common/microos/x86_64/' found.
Resolving package dependencies...
Nothing to do.
```

Reply:
Ok, so it sounds like it only needs to be installed as a workaround for that issue. In that case it should probably only do the `zypper_call` in the upstream script case and link the issue (maybe even softfail)

Reply:
Makes sense.